rise-storage.html

```diff
@@ -643,46 +643,14 @@
     /**
      * Processes a Storage file.
      */
-    _handleStorageFile: function(resp) {
-      var file = {},
-        etag = null;
-
-      // File in the root of the bucket.
-      if (resp.selfLink !== undefined) {
-        etag = resp.etag;
-      }
-      // File in a folder.
-      else if ((resp.files !== undefined) && (resp.files.length > 0)) {
-        etag = resp.files[0].etag;
-      }
+    _handleStorageFile: function() {
+      var file = {};
 
       file.url = this._fileUrl;
-
-      if (this._isLoading) {
-        this._files.push({
-          "name": this._getFileNameFromUrl(file.url),
-          "etag": etag
-        });
-
-        file.added = true;
-
-        this._isLoading = false;
-      }
-      else {
-        // File hasn't changed.
-        if (this._files[0].etag === etag) {
-          file.changed = false;
-        }
-        else {
-          this._files[0].etag = etag;
-          file.changed = true;
-        }
-      }
-
+      file.changed = true;
```

Comment:
@stulees I looked at the widgets and they are expecting the file.changed. That will only be needed when the widget encounter a problem then request the metadata to the component. If everything is fine with the metadata it should fire the event with the file.changed. 

I have also implemented something more robust. If the it is loading it will instead send a file.added flag. Please review. cheers